atmos.yaml

```
@@ -389,6 +389,30 @@ settings:
+        - "./README.md.gotmpl"
+      # The final README
+      output: "README.md"
+      terraform:
+        enabled: true
+        format: "markdown"
+        show_providers: false
+        show_inputs: true
+        show_outputs: true
+        sort_by: "name"
+        hide_empty: false
+        indent_level: 2  
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)